private/extensions/node.bzl

```diff
@@ -1,9 +1,99 @@
-"repositories for nodejs"
+"node"
```

Comment:
@omBratteng fyi, if you are updating node packages they are now in private/extensions/node.bzl

Reply:
👍